src/m_menu.c

```diff
@@ -2039,7 +2038,7 @@ void M_Setup(int choice)
 #define CR_TITLE  CR_GOLD
 #define CR_SET    CR_GREEN
 #define CR_ITEM   CR_NONE
-#define CR_HILITE CR_ORANGE
+#define CR_HILITE CR_NONE //CR_ORANGE
```

Comment:
@fabiangreffrath I removed the translation to `CR_ORANGE` in the setup menu and added `CR_BRIGHT` using `v_trans` code for consistency with the main menu. What do you think?

Reply:
Could you leave the macro intact so that `#define CR_HILITE CR_BRIGHT`? I don't seem to understand this change.